docs/src/examples/juliatypes.md

```diff
@@ -146,8 +146,8 @@ savefigs("ggplot", ans) # hide
 Creating a `Table` from a `DataFrame` will write it as expected.
 
 ```@example pgf
-using RDatasets
-df = dataset("datasets", "iris") # load the dataset
+using DataFrames, CSV
+df = CSV.read(joinpath(Pkg.dir("DataFrames"), "test/data/iris.csv"));
```

Comment:
Can't we just copy paste the dataset to this repo? `Pkg.dir` is deprecated on 0.7 and I'm not sure what the replacement will be.

Reply:
I experimented with that, but the path the examples get is whatever `pwd()` was then the tests were being run, so that is very brittle. Unless one wants to use a `@__DIR__` in the package. Can't we just open an issue and wait for the replacement of `Pkg.dir()`, and fix it then?